packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts

```diff
@@ -71,8 +73,68 @@ export class HorizontalBarChart extends FASTElement {
     this.hydrateLegends();
   }
 
+  public enableTabbability() {
+    requestAnimationFrame(() => {
+      const legendContainer = this.shadowRoot?.getElementById('legend-div') as HTMLElement;
+      if (legendContainer) {
+        const legendButtons = tabbable.tabbable(legendContainer);
+        let currentLegendIndex = 0;
+
+        legendContainer.addEventListener('keydown', (event: KeyboardEvent) => {
+          // Only handle Arrow keys for legend buttons
+          if (legendButtons.length > 0) {
+            currentLegendIndex = this.handleArrowNavigation(event, legendButtons, currentLegendIndex, this._isRTL);
+          }
+        });
+      }
+      // Handle navigation for tabbable elements inside root div
+      const rootDiv = this.shadowRoot?.getElementById('root-div');
+      if (rootDiv) {
+        const tabbableElements = tabbable.tabbable(rootDiv);
+        let currentIndex = 0;
+
+        rootDiv.addEventListener('keydown', (event: KeyboardEvent) => {
+          if (tabbableElements.length > 0 && event.target && tabbableElements.includes(event.target as HTMLElement)) {
+            currentIndex = this.handleArrowNavigation(event, tabbableElements, currentIndex, this._isRTL);
+          }
+        });
+      }
+    });
+  }
+  private handleArrowNavigation(
+    event: KeyboardEvent,
+    elements: tabbable.FocusableElement[],
+    currentIndex: number,
+    isRTL: boolean,
+  ): number {
+    let nextIndex = currentIndex;
+
+    // Handle ArrowRight and ArrowLeft
+    switch (event.key) {
+      case 'ArrowRight':
+        event.preventDefault();
+        nextIndex = isRTL
+          ? (currentIndex - 1 + elements.length) % elements.length
+          : (currentIndex + 1) % elements.length;
+        break;
+
+      case 'ArrowLeft':
+        event.preventDefault();
+        nextIndex = isRTL
+          ? (currentIndex + 1) % elements.length
+          : (currentIndex - 1 + elements.length) % elements.length;
+        break;
+
+      default:
+        return currentIndex;
+    }
+
+    // Focus the next element
+    elements[nextIndex]?.focus();
+    return nextIndex;
+  }
   public renderChart() {
-    const chartContainerDiv = d3Select(this.chartContainer);
+    const chartContainerDiv = d3Select(this.chartContainer).attr('id', 'root-div');
```

Comment:
use the chartContainer class variable directly. no need to set the id here.

Reply:
That class variables are not tied to the shadowDOM.
The shadowRoot contains the shadow DOM, but it does not store references to the DOM elements bound using ref.

Reply:
`chartContainer` is a reference to the element here: https://github.com/microsoft/fluentui/blob/bb66e2f89d1ba9e02b44644241cbdf995100146e/packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.template.ts#L13